Import Contents: CSV Template sets a test wrong value for Key/Value fields

Reproduced in demo.dotcms.com site. 

Support Ticket: https://my.dotcms.com/tickets/detail.dot?id=0ac83e9c-4edf-487e-ab95-0351ec75b4c8

Steps:

1. Add a Key/Value field to Blog Structure. For this test, it's called "GPS Coordinates".
2. Go to Import Contents portlet and download a CSV Sample file. 

Contents of CSV file: 

```
Host,Blog Headline,URL Title,Author,Publish,Body,Topic,Tags,CommentsCount,GPS coordinates
Host/Folder Identifier,XXX,XXX,XXX,MM/dd/yyyy hh:mm aa,XXX,Category Unique Key,XXX,XXX,XXX
```

The GPS Coordinates field has a value of "XXX" on this file, so any attempt to add a Key/Value field here will fail if it doesn't have a JSON-formatted value, like: 

```
"{'test':'1234'}"
``` 

Importing contents will fail because of the unformatted field and, for older releases where Key/Value fields were imported anyways, the Reindex will fail with messages in logfiles like the following: 

````
Caused by: com.dotcms.repackage.com.google.gson.JsonParseException: The JsonDeserializer MapTypeAdapter failed to deserialize json object -85.10968 given the type java.util.LinkedHashMap<java.lang.String, java.lang.String>
at com.dotcms.repackage.com.google.gson.JsonDeserializerExceptionWrapper.deserialize(JsonDeserializerExceptionWrapper.java:64) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationVisitor.invokeCustomDeserializer(JsonDeserializationVisitor.java:92) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationVisitor.visitUsingCustomHandler(JsonDeserializationVisitor.java:80) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.ObjectNavigator.accept(ObjectNavigator.java:101) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationContextDefault.fromJsonPrimitive(JsonDeserializationContextDefault.java:85) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationContextDefault.deserialize(JsonDeserializationContextDefault.java:56) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:551) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:498) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:467) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:417) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotmarketing.portlets.structure.model.KeyValueFieldUtil.JSONValueToHashMap(KeyValueFieldUtil.java:48) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESMappingAPIImpl.loadFields(ESMappingAPIImpl.java:543) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESMappingAPIImpl.toMap(ESMappingAPIImpl.java:262) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESContentletIndexAPI.indexContentletList(ESContentletIndexAPI.java:327) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESContentletIndexAPI.access$100(ESContentletIndexAPI.java:43) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESContentletIndexAPI$1.run(ESContentletIndexAPI.java:297) ~[dotcms_3.3_1bbd913.jar:?]
... 3 more
Caused by: java.lang.IllegalStateException: This is not a JSON Object.
at com.dotcms.repackage.com.google.gson.JsonElement.getAsJsonObject(JsonElement.java:83) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.MapTypeAdapter.deserialize(MapTypeAdapter.java:66) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.MapTypeAdapter.deserialize(MapTypeAdapter.java:33) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializerExceptionWrapper.deserialize(JsonDeserializerExceptionWrapper.java:51) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationVisitor.invokeCustomDeserializer(JsonDeserializationVisitor.java:92) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationVisitor.visitUsingCustomHandler(JsonDeserializationVisitor.java:80) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.ObjectNavigator.accept(ObjectNavigator.java:101) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationContextDefault.fromJsonPrimitive(JsonDeserializationContextDefault.java:85) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.JsonDeserializationContextDefault.deserialize(JsonDeserializationContextDefault.java:56) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:551) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:498) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:467) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotcms.repackage.com.google.gson.Gson.fromJson(Gson.java:417) ~[dot.tika-app-1.3_3.jar:1.3]
at com.dotmarketing.portlets.structure.model.KeyValueFieldUtil.JSONValueToHashMap(KeyValueFieldUtil.java:48) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESMappingAPIImpl.loadFields(ESMappingAPIImpl.java:543) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESMappingAPIImpl.toMap(ESMappingAPIImpl.java:262) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESContentletIndexAPI.indexContentletList(ESContentletIndexAPI.java:327) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESContentletIndexAPI.access$100(ESContentletIndexAPI.java:43) ~[dotcms_3.3_1bbd913.jar:?]
at com.dotcms.content.elasticsearch.business.ESContentletIndexAPI$1.run(ESContentletIndexAPI.java:297) ~[dotcms_3.3_1bbd913.jar:?]
... 3 more
``` 

This is related to https://github.com/dotCMS/core/issues/8315 however, for bad contents, the Priority Level is set to 40 + the amount of attempts for reindexing these contents (defaults to 5).

--- a/src/com/dotmarketing/portlets/contentlet/action/ImportContentletsAction.java
+++ b/src/com/dotmarketing/portlets/contentlet/action/ImportContentletsAction.java
@@ -328,19 +328,21 @@ public class ImportContentletsAction extends DotPortletAction {
 			Field field = fields.get(i);
 			if (ImportUtil.isImportableField(field)) {
 				if (field.getFieldType().equals(Field.FieldType.DATE.toString())) {
-					out.print("MM/dd/yyyy");
+					out.print("\"MM/dd/yyyy\"");
 				}        	
 				else if (field.getFieldType().equals(Field.FieldType.DATE_TIME.toString())) {
-					out.print("MM/dd/yyyy hh:mm aa");
+					out.print("\"MM/dd/yyyy hh:mm aa\"");
 				}         	
 				else if (field.getFieldType().equals(Field.FieldType.TIME.toString())) {
-					out.print("hh:mm aa");
+					out.print("\"hh:mm aa\"");
 				}else if (field.getFieldType().equals(Field.FieldType.HOST_OR_FOLDER.toString())) {
-						out.print("Host/Folder Identifier");
+						out.print("\"Host/Folder Identifier\"");
 				}else if (field.getFieldType().equals(Field.FieldType.CATEGORY.toString())) {
-					out.print("Category Unique Key");
-				} else {
-					out.print("XXX");
+					out.print("\"Category Unique Key\"");
+				}else if (field.getFieldType().equals(Field.FieldType.KEY_VALUE.toString())){
+				    out.print("\"{'key1':'value','key2':'value2'}\"");
+				}    else {
+					out.print("\"XXX\"");
 				}
 				if(i < fields.size() - 1) 
 					out.print(",");

